pettingzoo/utils/wrappers/supersuit/utils/base_aec_wrapper.py

```diff
@@ -0,0 +1,115 @@
+from pettingzoo.utils.wrappers import OrderEnforcingWrapper as PettingzooWrap
+from pettingzoo import AECEnv, ParallelEnv
+from pettingzoo.utils.env import ActionType, AgentID
+from gymnasium.spaces import Space
+
+
+class BaseWrapper(PettingzooWrap):
+    def __init__(self, env: AECEnv | ParallelEnv):
+        """
+        Creates a wrapper around `env`. Extend this class to create changes to the space.
```

Comment:
Also this looks incorrect saying `class BaseWrapper(PettingZooWrap)`, why would it use OrderEnforcingWrapper as the base class? And I'd honestly have to think more about what the base class should be, it's tough cause a lot of the functions like step() and reset() have different return types and aren't uniform between parallel and AEC.

Reply:
I haven't really looked into the wrappers for the moment, this is just as it was on SuperSuit I can try to find a better base class for BaseWrapper